angular/.gitignore

```diff
@@ -1,48 +1,5 @@
-# See http://help.github.com/ignore-files/ for more about ignoring files.
-
-# compiled output
 /dist
-/storybook-static
-/tmp
-documentation.json
-/out-tsc
-# Only exists if Bazel was run
-/bazel-out
-
-# dependencies
 /node_modules
-
-# profiling files
-chrome-profiler-events*.json
-speed-measure-plugin*.json
-
-# IDEs and editors
```

Comment:
we dont want to commit editor stuff, so should keep these

Reply:
so I'm wondering why we would need this when it's never existed in other gitignores.

Reply:
@RCopeland and others have personal editor configs setup that we don't want to accidentally commit. I would like you to research why this wasn't a problem for us before.